xroad-catalog-collector/src/main/java/fi/vrk/xroad/catalog/collector/util/OrganizationUtil.java

```diff
@@ -823,29 +696,23 @@ public static JSONArray getDataByIds(ClientType clientType, List<String> guids,
                             + url,
                     "500");
             catalogService.saveErrorLog(errorLog);
-            log.error("KeyManagementException occurred when fetching organizations with from url {}",
-                    url);
+            log.error("KeyManagementException occurred when fetching organizations with from url {}", url);
         } catch (Exception e) {
-            log.error("Exception occurred when fetching organization data: "
-                    + e.getMessage());
+            log.error("Exception occurred when fetching organization data: " + e.getMessage());
             ErrorLog errorLog = ErrorLog.builder()
                     .created(LocalDateTime.now())
                     .message("Exception occurred when fetching organization data: "
                             + e.getMessage())
                     .code("500")
-                    .xRoadInstance(clientType.getId()
-                            .getXRoadInstance())
+                    .xRoadInstance(clientType.getId().getXRoadInstance())
                     .memberClass(clientType.getId().getMemberClass())
                     .memberCode(clientType.getId().getMemberCode())
                     .groupCode(clientType.getId().getGroupCode())
-                    .securityCategoryCode(clientType.getId()
-                            .getSecurityCategoryCode())
+                    .securityCategoryCode(clientType.getId().getSecurityCategoryCode())
```

Comment:
The `securityCategoryCode` could be removed since support for security categories was removed already earlier:

https://github.com/nordic-institute/X-Road/pull/1615

Reply:
Removed.